docs/long_desc.rst

```diff
@@ -3,11 +3,27 @@ BanditPAM: A state-of-the-art, high-performance k-medoids algorithm
 
 Introduction
 ------------
-Clustering algorithms such as k-means are ubiquitous in modern data science applications. Despite its popularity, however, k-means has several drawbacks. K-means only supports certain distance metrics, and its cluster centers can lack interpretability. Alternatively, k-medoids uses the points in the dataset itself -- the medoids -- as cluster centers. This enables interpretability of the cluster centers. Furthermore, k-medoids supports arbitrary dissimilarity functions in place of a distance metric.
+Clustering algorithms are ubiquitous in modern data science applications. Compared to the commonly used k-means clustering, k-medoids clustering 
+uses the points in the dataset itself -- the medoids -- as cluster centers and supports arbitrary dissimilarity functions
+in place of a distance metric. This enables greater interpretability of the cluster centers and the clustering of structured 
+objects. Despite its advantages, k-medoids clustering is less popular than k-means due to its computational cost. 
+The k-medoids algorithms such as PAM and FastPAM scale quadratically in the dataset size in each iterations
+but they are still significantly slower than k-means, which scales linearly in dataset size in each iteration. We introduce BanditPAM, 
+a randomized algorithm inspired by techniques from multi-armed bandits, that runs significantly faster than prior k-medoids algorithms and achieves the same clustering results. 
```

Comment:
`that scales almost linearly with dataset size and runs significantly faster than prior algorithms while still matching the best prior algorithms in clustering quality.`

Reply:
Let's not bury the lede. Let's move the sentence starting with "we present..." before the sentence starting "despite its advantages...", so we emphasize the value of the package early.

At the end of the new paragraph ending in "significantly slower than k-means, which scales linearly in dataset size in each iteration.", add a sentence saying "This algorithm, BanditPAM, matches k-means in complexity and is significantly faster than prior state-of-the-art."